src/main/java/com/budgetapp/controllers/ExpensesController.java

```diff
@@ -42,4 +44,14 @@ public ResponseEntity<Expense> saveExpenses(@RequestBody final Expense expense)
         final var createExpenses = expensesService.addExpenses(expense);
         return ResponseEntity.status(HttpStatus.CREATED).body(createExpenses);
     }
+
+    @GetMapping(path = "/{user_id}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
This doesn't follow Java naming conventions

```suggestion
    @GetMapping(path = "/{userId}", produces = MediaType.APPLICATION_JSON_VALUE)
```